### ISSUE TITLE
Focus state removed from btn-link:focus ruleset

### ISSUE THREAD

**Issue by derick-montague** [twbs/bootstrap#29891]
## How to reproduce
1. Set focus on an element with the `.btn-link` class

## Expected Behavior
There will be a visual indicator that the element has received focus.

## Actual Behavior
The element does not have a visual indicator for the focus state.

## Other Info
- Operating system and version: Any
- Browser and version: Any

## Cause
Focus states are set using the `box-shadow` property which has a value of `none` in the `.btn-link:focus` declaration block.
